recipe/yum_requirements.txt

```diff
@@ -0,0 +1,16 @@
+liburiparser-devel
```

Comment:
Thanks @ocefpaf I'm inside the Docker container right now, so that should indeed prove helpful in the future.

However, several of the requirements don't seem to be available through yum. I downloaded the filelist of one of the repo's (mirror.cj2.nl) my machine is connected to and in their filelist some dependencies are available, but only for version 7.2

- uriparser
- libmspack
- zziplib
- graphite2
- log4cxx

And the following don't yet seem available at all:

- xsltproc (though we might have libxslt for that)
- botan
- potrace